crates/sui-graphql-rpc/src/types/object.rs

```diff
@@ -236,7 +248,8 @@ impl Object {
             .extend()
     }
 
-    /// The balances of all coin types owned by the object. Coins of the same type are grouped together into one Balance.
+    /// The balances of all coin types owned by the object.
```

Comment:
Why the change in whitespace here? Generally, the whitespace in these docs is tracking a little strangely for me (the reason I bring it up is that these carry through into the docs, at least on GraphiQL, unfortunately).  Can we stick to adding line wraps at column 100, and adding a blank line between paragraphs?